src/NLog/Layouts/Layout.cs

```diff
@@ -126,6 +126,31 @@ public static Layout CreateFromString(string layoutText)
             return new SimpleLayout(layoutText, ConfigurationItemFactory.Default);
         }
 
+        /// <summary>
+        /// Implicitly converts the specified string to a <see cref="SimpleLayout"/>.
+        /// </summary>
+        /// <param name="layoutText">The layout string.</param>
+        /// <param name="throwConfigExceptions">Whether <see cref="NLogConfigurationException"/> should be thrown on parse errors.</param>
```

Comment:
Please add that if set to false, then invalid renderers will be replace with a space.

Reply:
Fixed